packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -251,6 +251,46 @@ describe("Runtime", () => {
 				assert.strictEqual(containerRuntime.flushMode, FlushMode.Immediate);
 			});
 
+			it("Process empty batch", async () => {
+				const containerRuntime = await ContainerRuntime.loadRuntime({
+					context: getMockContext({
+						"Fluid.Container.enableOfflineLoad": true,
+					}) as IContainerContext,
+					registryEntries: [],
+					existing: false,
+					runtimeOptions: {
+						flushMode: FlushMode.TurnBased,
+					},
+					provideEntryPoint: mockProvideEntryPoint,
+				});
+				let callsToEnsure = 0;
+				(containerRuntime as any).ensureNoDataModelChanges = () => {
+					callsToEnsure++;
+				};
```

Comment:
For future reference you can use Sinon for this too:
```suggestion
				const ensureSpy = Sinon.spy(containerRuntime, "ensureNoDataModelChanges");
```
Then you can assert `ensureSpy.callCount` is 1.  It has lots of other handy functions on there for asserting how the function was called.